frame/support/test/tests/pallet.rs

```diff
@@ -1906,14 +1906,26 @@ fn assert_type_all_pallets_without_system_reversed_is_correct() {
 
 #[test]
 fn test_storage_alias() {
+	use frame_support::Twox64Concat;
+
 	#[frame_support::storage_alias]
 	type Value<T: pallet::Config>
 	where
 		<T as frame_system::Config>::AccountId: From<SomeType1> + SomeAssociation1,
 	= StorageValue<pallet::Pallet<T>, u32, ValueQuery>;
 
+	#[frame_support::storage_alias]
+	type SomeCountedStorageMap<T: pallet2::Config>
+	where
+		<T as frame_system::Config>::AccountId: From<SomeType1> + SomeAssociation1,
+	= CountedStorageMap<pallet2::Pallet<T>, Twox64Concat, u8, u32>;
+
 	TestExternalities::default().execute_with(|| {
 		pallet::Value::<Runtime>::put(10);
 		assert_eq!(10, Value::<Runtime>::get());
+
+		pallet2::SomeCountedStorageMap::<Runtime>::insert(10, 100);
```

Comment:
Could you also check that the returned meta-data is identical to the non-alias version?

Reply:
> Could you also check that the returned meta-data is identical to the non-alias version?

Care to elaborate?

Reply:
done in https://github.com/paritytech/substrate/pull/13366/commits/79378d1000f4858aa37c1783e611cca4a29e25a4